source/gui/settingsDialogs.py

```diff
@@ -35,6 +35,18 @@
 	updateCheck = None
 import inputCore
 
+#Choices for numbers combo box in Voice settings.
+
+DIGIT_CHOICES = [
+	# Translators: Choice in a combo box for speaking numbers by word.
+	_("Words"),

```

Comment:
Full numbers or raw numbers, although I'd rather go with full numbers.



On 9/7/2016 6:40 PM, James Teh wrote:
>
> In source/gui/settingsDialogs.py 
> <https://github.com/nvaccess/nvda/pull/5930#discussion_r77930043>:
>
> > @@ -35,6 +35,18 @@
> >  	updateCheck = None
> >  import inputCore
> >
> > +#Choices for numbers combo box in Voice settings.
> > +
> > +DIGIT_CHOICES = [
> > +	# Translators: Choice in a combo box for speaking numbers by word.
> > +	_("Words"),
>
>   * "Controlled by synthesizer" is quite long, though. I would have
>     thought most synths did full numbers by default. I'm also
>     concerned some users won't really understand what this means in
>     practice.
>   * That said, "Words" is potentially confusing, since double and
>     triple digits technically produce words. Maybe we could call it
>     "Full" or "Full numbers"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/5930/files/7507485b589cdf070e32b05a0a87e4de96d812af..13a7a7fdaefae133b7c7e474cba5cb98d4be1b4b#r77930043>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFGivRs51_euGP1LlGiX_hH1_rfjSVPHks5qn1l_gaJpZM4IYz-c>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194